megamek/src/megamek/server/Server.java

```diff
@@ -26667,7 +26685,30 @@ else if (secondaryEffects) {
                 }
                 // check for possible explosion
                 int fuelroll = Compute.d6(2);
+                r = new Report(9120);
+                r.subject = aero.getId();
                 if (fuelroll > boomTarget) {
```

Comment:
I realize this is existing code, but why `>` and not `>=` like elsewhere?

Reply:
Kind of wondered why the previous author did it that way too, to be honest - I'd rather have the actual numbers from SO show up in the code instead of the offset-by-1 needed to use >.  Fixed.